docker/server/src/main/docker/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN set -eux; \
 
 COPY deephaven-wheel/ /deephaven-wheel
 RUN set -eux; \
-    python -m pip install -q --no-index --no-cache-dir /deephaven-wheel/*.whl; \
+    python -m pip install -q --no-cache-dir /deephaven-wheel/*.whl; \
```

Comment:
For final version, we shouldn't be checking in this change. If deephaven-plugin remains external, we'd need to update the deephaven base images as appropriate.